.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "semi": true,
+  "singleQuote": true
```

Comment:
**two question:**

- what style do you prefer ?
- should I run format on whole codebase, or you wanna keep it to be re-formatted gradually

Reply:
Thank you for asking! 😃I usually go with:
```json
{
    "trailingComma": "none",
    "tabWidth": 4,
    "semi": true,
    "singleQuote": true,
    "printWidth": 120
}
```

What do you think?

Reply:
I've quite opposite style :)
anyways these things are handled by robots, so I don't care :) 

going to apply your setup. thx!

Reply:
one suggestion though, I'd change "printWidth" to 100 or 80. 120 long lines is really long... :D

Reply:
:D I'm really open, tbh. As you said, it's _automated_, no need to worry about what goes where. So, yup, 100, 80, I have no preference. Your call here. 👍